### PR TITLE
Update CLI reference for show application names

### PR DESCRIPTION
**Pull request by agrawalkaushik**

### CODE REVIEW

File: docs/cli_reference.md [hunk 1]
```diff
@@ -3426,14 +3426,27 @@ Wed 2020-04-22 16:06:43 UTC
...
+ google-drive                   0               27   2021-04-14 13:58:09   2021-04-14 13:58:09
+```
+
+The various columns are as follows:
+
+| Column | Description                      | 
+| ------ | ---------------------------------|
+| Application Name | The name of the identified application. |
+| Session Count | The number of active sessions for the application. | 
+| Ip Tuple Count | The number of address and ports associated with the application. |
```
Comment (ptimmons) [inline docs/cli_reference.md:3447]: s/address/addresses

File: docs/cli_reference.md [hunk 2]
```diff
@@ -3426,14 +3426,27 @@ Wed 2020-04-22 16:06:43 UTC
...
+| Date Discovered | The time at which the router first learned this application. Restarting the router or deletion of the application will reset the time.  |
```
Comment (ptimmons) [inline docs/cli_reference.md:3448]: Restarting the router or deleting the application

**Review by ptimmons — CHANGES_REQUESTED** [review]
Small nits.

**Review by MichaelBaj — APPROVED** [review]

**Review by robert-patterson — DISMISSED** [review]

File: docs/cli_reference.md (hunk 1 above)
Comment (agrawalkaushik) [reply docs/cli_reference.md]: done

File: docs/cli_reference.md (hunk 2 above)
Comment (agrawalkaushik) [reply docs/cli_reference.md]: done

**Review by ptimmons — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**